app/views/candidates/registrations/account_checks/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<%=
+  render \
```

Comment:
More FYI: pretty sure this can just be written as `render 'form', account_check: @account_check`